docs/snippets/ov_infer_request.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+#! [import]
+import openvino.runtime as ov
+#! [import]
+
+core = ov.Core()
+model = core.read_mode("model.xml")
+compiled_model = core.compile_model(model, "AUTO")
+
+#! [create_infer_request]
+infer_request = compiled_model.create_infer_request()
+#! [create_infer_request]
+
+#! [sync_infer]
+infer_request.infer()
+#! [sync_infer]
+
+#! [async_infer]
+infer_request.start_async()
+#! [async_infer]
+
+#! [wait]
+infer_request.wait()
+#! [wait]
+
+#! [wait_for]
+infer_request.wait_for(10)
+#! [wait_for]
+
+#! [set_callback]
+def callback(request):
+    request.start_async()
```

Comment:
```suggestion
def callback(request, userdata):
    request.start_async()
```
It needs two arguments

Reply:
Thanks!